### DISCUSSION TITLE
[question] how build on Fedora 36 with pipewire and wayland?

### DISCUSSION THREAD

**Discussion by poulpoulsen** [DISTRHO/Cardinal#426] [Q&A]
Hello,
just one question:
is it possible to build with pipewire and wayland?
if so, where could i read a bit about it?

thanks for this wonderful package!

Regards
Poul

**Comment by falkTX** [comment]
this is not an issue, so moved to discussions.
building does not make a difference with jack vs pipewire, and wayland is never used.

I think Cardinal is on flathub somewhere, but I dont maintain that.

for building docs see https://github.com/DISTRHO/Cardinal/blob/main/docs/BUILDING.md

**Reply by poulpoulsen** [reply]
hello,
i just try to build it, but make command stops with this message:
Makefile:105: *** OpenGL dependency not installed/available.  Schluss.

line 105 contains
ifneq ($(HAVE_OPENGL),true)
$(error OpenGL dependency not installed/available)
endif

in makefile.base.mk

ifeq ($(MACOS_OR_WASM_OR_WINDOWS),true)
HAVE_OPENGL = true
else
HAVE_OPENGL  = $(shell $(PKG_CONFIG) --exists gl && echo true)
HAVE_DBUS    = $(shell $(PKG_CONFIG) --exists dbus-1 && echo true)
HAVE_X11     = $(shell $(PKG_CONFIG) --exists x11 && echo true)
HAVE_XCURSOR = $(shell $(PKG_CONFIG) --exists xcursor && echo true)
HAVE_XEXT    = $(shell $(PKG_CONFIG) --exists xext && echo true)
HAVE_XRANDR  = $(shell $(PKG_CONFIG) --exists xrandr && echo true)
endif
a package called gl is installed.
a package called x11 is also there.

what is going wrong?

Regards
Poul

**Reply by poulpoulsen** [reply]
Wow, after a complete refresh and pull the dependencies now it works out of the box.
Thanks again for the great package.
in opposite of the original Rack2 the audio connection works.
in Rack2 compiling without a jack-server does not work.
therefore with pipewire under fedora Rack2 does not work.
But yours!!
